plugins/modem-manager/fu-mm-device.c

```diff
@@ -625,6 +625,24 @@ fu_mm_device_at_cmd_cb(FuDevice *device, gpointer user_data, GError **error)
 	}
 	fu_dump_bytes(G_LOG_DOMAIN, "read", at_res);
 	buf = g_bytes_get_data(at_res, &bufsz);
+
+	/*
+	 * the first time the modem returns may be the command itself with one \n missing.
+	 * this is because the modem AT has enabled echo
+	 */
+	if (g_strrstr(buf, helper->cmd) != NULL && bufsz == strlen(helper->cmd) + 1) {
+		at_res = fu_io_channel_read_bytes(self->io_channel,
```

Comment:
don't we need a g_bytes_unref(at_res) here?

Reply:
sorry, I was negligent.

Thanks.